SU2_CFD/include/solvers/CScalarSolver.inl

```diff
@@ -268,6 +268,14 @@ void CScalarSolver<VariableType>::Upwind_Residual(CGeometry* geometry, CSolver**
         if (implicit) Jacobian.UpdateBlocks(iEdge, iPoint, jPoint, residual.jacobian_i, residual.jacobian_j);
       }
 
+      /*--- Applying convective flux correction to negate the effects of flow divergence ---*/
+      for(iVar=0; iVar<nVar; iVar++){
+        FluxCorrection_i = GetNodes()->GetSolution(iPoint, iVar) * EdgeMassFlux;
+        FluxCorrection_j = GetNodes()->GetSolution(jPoint, iVar) * EdgeMassFlux;
+
+        LinSysRes(iPoint, iVar) -= FluxCorrection_i;
+        LinSysRes(jPoint, iVar) += FluxCorrection_j;
+      }
```

Comment:
Why is this needed?

Reply:
I'm a bit confused by the modification of the correction compared to the massflux scheme directly above it and what in the end the complete corrected numerical scheme is. It would be good to write it out. 
 
  ```
   if (ReducerStrategy) {
        EdgeFluxes.SetBlock(iEdge, residual);
        if (implicit) Jacobian.SetBlocks(iEdge, residual.jacobian_i, residual.jacobian_j);
      } else {
        LinSysRes.AddBlock(iPoint, residual);
        LinSysRes.SubtractBlock(jPoint, residual);
        if (implicit) Jacobian.UpdateBlocks(iEdge, iPoint, jPoint, residual.jacobian_i, residual.jacobian_j);
      }
```

Reply:
[Bounded_Scalar_Scheme.pdf](https://github.com/su2code/SU2/files/10074743/Bounded_Scalar_Scheme.pdf)
I wrote a short explanation of how the bounded scalar scheme was implemented and what it does when used in the venturi test case. Let me know if you have any suggestions for improvement.